webpack.dev-site.config.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const merge = require('webpack-merge');
+const {
+  TerraDevSite,
+  TerraDevSiteEntrypoints,
+  DirectorySwitcherPlugin,
+  LocalPackageAliasPlugin,
+} = require('terra-dev-site');
```

Comment:
Is this going to result in a cyclical dependency at all?

Reply:
no, but it does result in an unmet peer.